config/v1/types_infrastructure.go

```diff
@@ -369,9 +403,12 @@ type AWSPlatformStatus struct {
 
 	// resourceTags is a list of additional tags to apply to AWS resources created for the cluster.
 	// See https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html for information on tagging AWS resources.
-	// AWS supports a maximum of 50 tags per resource. OpenShift reserves 25 tags for its use, leaving 25 tags
+	// In previous versions, OpenShift reserved 25 tags for its use, leaving 25 tags
 	// available for the user.
+	// ResourceTags field is immutable.
+	// To update the list of tags, contact support.
 	// +kubebuilder:validation:MaxItems=25
+	// +kubebuilder:deprecatedversion:warning="ResourceTags in .Status is deprecated, use .Spec.PlatformSpec.AWS.ResourceTags"
```

Comment:
This appears to preclude having common API behaviors like removing an item from .spec functioning.